config/application.rb

```diff
@@ -86,8 +86,9 @@ class Application < Rails::Application
     # These defaults can be overridden per-environment if needed
     config.start_of_unique_links_only_intake = Time.find_zone('America/Los_Angeles').parse('2023-01-24 09:59:59')
     config.start_of_open_intake = Time.find_zone('America/Los_Angeles').parse('2023-01-31 09:59:59')
+    config.tax_deadline = Time.find_zone('America/New_York').parse('2023-04-18 23:59:59')
```

Comment:
i know this is today but maybe (maybe??) we can in future just change these dates when we want to update the banners but who knows it might have been a big waste of time to make it this smart and maybe i should have just replaced the dang thing instead of adding more helper methods